.goreleaser.yml

```diff
@@ -0,0 +1,102 @@
+## Bitnami Sealed Secrets
+
+release:
+  # Comment out to enable publishing
+  disable: true
+
+builds:
+- binary: controller
+  main: ./cmd/controller/
+  env:
+  - CGO_ENABLED=0
+  goarch:
+  - amd64
+  - arm
+  - arm64
+  goos:
+  - linux
+  - darwin
+  - windows
+  ldflags: -X main.VERSION={{.Version}}
+- binary: kubeseal
+  main: ./cmd/kubeseal/
+  env:
+  - CGO_ENABLED=0
+  goarch:
+  - amd64
+  - arm
+  - arm64
+  goos:
+  - linux
+  - darwin
+  - windows
+  ldflags: -X main.VERSION={{.Version}}
+
+brew:
+  github:
+    owner: bitnami-labs
+    name: sealed-secrets
+  homepage: "https://github.com/bitnami-labs/sealed-secrets/"
+  description: "A Kubernetes controller and tool for one-way encrypted Secrets"
+  skip_upload: true
+  # Replace with existing brew formula
+  test: |
+    system "#{bin}/program --version"
+
+dockers:
+- goarch: amd64
+  goos: linux
+  binaries:
+  - controller
+  dockerfile: ./docker/Dockerfile
+  image_templates:
+  - bitnami-labs/sealed-secrets-controller:latest
+  - bitnami-labs/sealed-secrets-controller:amd64-latest
+  - bitnami-labs/sealed-secrets-controller:amd64-{{ .Version }}
+  # Comment below out to push to configured repository
+  skip_push: true
+- goarch: arm64
+  goos: linux
+  binaries:
+  - controller
+  dockerfile: ./docker/Dockerfile
+  image_templates:
+  - bitnami-labs/sealed-secrets-controller:arm64-latest
+  - bitnami-labs/sealed-secrets-controller:arm64-{{ .Version }}
+  # Comment below out to push to configured repository
+  skip_push: true
+- goarch: arm
+  goarm: 6
+  goos: linux
+  binaries:
+  - controller
+  dockerfile: ./docker/Dockerfile
+  image_templates:
+  - bitnami-labs/sealed-secrets-controller:armv6-latest
+  - bitnami-labs/sealed-secrets-controller:armv6-{{ .Version }}
+  # Comment below out to push to configured repository
+  skip_push: true
+
+archive:
+  replacements:
```

Comment:
Thanks, good point.

> Also (note to self): This will change the install instructions. We need to time an update to the README to uncompress the new tarball. Perhaps we just link to github releases and don't give cut+paste commands.

Couldn't help but think through this a bit. 

Perhaps we can simplify the README installation process right down if:
- commit `controller.yaml`'s to [/examples](https://github.com/bitnami-labs/sealed-secrets/tree/master/examples) with `latest` image. Serious users will pin versions/build/etc.
- remove/update `archive.replacements`. Not sure about arm64/aarch64 or armv7/armv6.
- CRD assumption, fold into controller.yaml (with kind: namespace, etc), TPR separate. 
- move Source/Test/Developing to separate CONTRIBUTING.md or DEVELOPING.md?

## Server Installation

```
# For clusters with RBAC
wget https://github.com/bitnami-labs/sealed-secrets/examples/controller.yaml
kubectl apply -f controller.yaml

# For clusters without RBAC
wget https://github.com/bitnami-labs/sealed-secrets/examples/controller-norbac.yaml
kubectl apply -f controller-norbac.yaml
```

## Client Installation
MacOS: 
```
brew install kubeseal
```
Linux:
```
# get latest release version
release=$(curl --silent "https://api.github.com/repos/bitnami-labs/sealed-secrets/releases/latest" | sed -n 's/.*"tag_name": *"\([^"]*\)".*/\1/p')

# Download release
wget https://github.com/bitnami-labs/sealed-secrets/releases/download/$release/sealed-secrets_$release_$(uname)_$(uname -m).tar.gz

# Extract release
tar -zxvf sealed-secrets_$release_$(uname)_$(uname -m).tar.gz

# Install in path (if required)
sudo install -m 755 kubeseal /usr/local/bin/kubeseal
```

Windows:
- download latest [release](https://github.com/bitnami-labs/sealed-secrets/releases/latest)
- extract archive

## Client usage 

(Existing docs is fine. I would just add fetching cert here)

### Optional one time process - save encryption cert to file

The public cert can be commited to git and supplied as an argument to `kubeseal` instead of requiring cluster access. For example: `kubeseal --cert mycert.pem`
```
# Save public cert to file (accessible controller)
kubeseal --fetch-cert > mycert.pem

# Save public cert to a file (kubectl access to secret)
kubectl get secret -n sealed-secrets sealed-secrets-key -o json | jq -r '.data | ."tls.crt"' > mycert.pem
```
### Creating a Sealed Secret

.... from [existing docs](https://github.com/bitnami-labs/sealed-secrets#usage)